.github/workflows/mac.yml

```diff
@@ -91,12 +91,20 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           path: ${{ github.workspace }}/${{ env.SDK_TAR }}
+          if-no-files-found: error
 
+      - name: Upload supported formats in Artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: supported_formats.log
+          path: ${{ github.workspace }}/build/stage/mac/supported_formats.log
+          if-no-files-found: error
+          
       - name: Create Release
         if: ${{ github.ref == 'refs/heads/master' }}
         uses: ncipollo/release-action@v1
         with:
           tag: mac-${{ steps.time.outputs.time }}-${{ github.run_number }}
           allowUpdates: true
-          artifacts: ${{ env.SDK_TAR }}
+          artifacts: ${{ env.SDK_TAR }}, supported_formats.log
```

Comment:
Why do we export `supported_formats.log` only in mac build? Also, it seems that it contains supported drivers from GDAL/OGR. Are there other drivers like PostGIS, XYZ which are not included in that log?

Reply:
postgis / xyz are QGIS providers, these are GDAL drivers. we run only on mac, because it *should* be same on all platforms (all same libs) and also you need to run in on the host (so you cannot run gdalinfo binary compiled for iOS on macOS, etc)